### ISSUE TITLE
Project Deploy - Large Projects - Zip64 Error

### ISSUE THREAD

**Issue by ChartisKathy** [DMS-Aus/Roam#379]
After clicking the "Publish Update" button on a 10MB project whose _data folder has imagery (27GB), I get this error - basically stating that the project is too large.
![image](https://user-images.githubusercontent.com/26241911/27667927-57e68864-5cc1-11e7-899c-baa5705e2749.png)

**Comment by NathanW2** [comment]
Yeah I'm not sure I would include 27Gb of data inside the _data folder
because it will then have to download the data.  That kind of thing just
has to be a manual install at the momment.

**Comment by ChartisKathy** [comment]
Agree transfers of large amounts of data should be avoided. Especially over the web.

Can an enhancement be made to exclude the _data folder for the Project Update button? Or potentially for a popup to come up prompting the user whether data should be part of the install / update?

Thanks mate.

**Comment by bzeeb** [comment]
We should catch this error and stop the user publishing projects that large, perhaps >2 GB? Rather arbitrary limit, where does zipfile 32bit library cut off?

Regardless though, would recommend moving that imagery file outside the project completely. We generally store ours at the project level eg. C:\IntraMaps Roam\projects\latest_aerial.ecw
